datacenter/dtr/2.0/high-availability/backups-and-disaster-recovery.md

```diff
@@ -86,6 +86,8 @@ $ tar -tf /tmp/backup.tar
 
 ## Restore DTR data
 
+Performing restore require restoration to the same release that the backups were 
```

Comment:
Can you please update the phrasing a bit? Something like "DTR data must be restored to the same release version where the backups were obtained." Or something like that. What do you think?